### ISSUE TITLE
Section headings without numbers

### ISSUE THREAD

**Issue by psibre** [psibre/2017-essv-authorkit#8]
One user reports that when compiling under Ubuntu 16.04, the section headings are not numbered, compare original (left) and recompiled (right):
![diff](https://cloud.githubusercontent.com/assets/1278639/21800291/4274bffc-d71d-11e6-8ef3-ff5eeb223722.png)

I can reproduce this with with TeXLive 2015 running on Ubuntu 16.04.

**Comment by psibre** [comment]
This appears to be related to the use of [titlesec](https://www.ctan.org/tex-archive/macros/latex/contrib/titlesec) in the `essv.sty` package file.

A simple **workaround** is to download the latest `titlesec.sty` from http://mirrors.ctan.org/macros/latex/contrib/titlesec/titlesec.sty into the paper directory, where it will be used instead of the one from the TeXLive installation.
Upgrading the titlesec may also work, if one has the required user privileges.

**Comment by seblemaguer** [comment]
FYI: this problem does not occur in 16.10

**Comment by psibre** [comment]
@seblemaguer Yes, I think it's only triggered by certain versions of `titlesec.sty`.
